stdlib/public/Cxx/libstdcxx/libstdcxx.h

```diff
@@ -1,7 +1,9 @@
+#include "version"
```

Comment:
@ADKaster if possible, (if this is a header visible on **Microsoft Windows**) I believe this include will break on that platform, since the `vcruntime.modulemap` marks that `version` header as **C++20** language standard **REQUIRED**.

With an error that looks something like the following:
```swift
// SomeApp.swift
import SomeCXXTarget
       ^~~~~~~~~~~~~
vcruntime.modulemap:696:10: error: module 'std.version' requires feature 'cplusplus20'
```

Perhaps this whole file can be guarded via `#if (__WIN32)`, if this header is even visible on windows in the first place.

Thanks!

Reply:
This header isn't used on Windows. We currently only support the platform-default C++ stdlib on each platform (there is an effort to change that in https://github.com/swiftlang/swift/pull/75589). On Windows that would be the Microsoft STL.

Reply:
Awesome, thanks @egorzhdan for the clarification!